documentation/src/main/asciidoc/topics/ref_server_realm_trust.adoc

```diff
@@ -1,18 +1,26 @@
-[id='trust_store_realm-{context}']
+[id='trust-store-realms_{context}']
 = Trust Store Realms
-Trust store realms use keystores that contain certificates, or certificate chains, that clients must present to connect to {brandname} Server.
+{brandname} Server can encrypt traffic using keystores that contain server and client certificates.
 
-[source,xml,options="nowrap",subs=attributes+]
-----
-include::config_examples/server_truststore_realm.xml[]
-----
+Keystores:: Contain server certificates that provide a {brandname} Server identity to clients.
+Trust stores:: Contain client certificates, or certificate chains, that allow {brandname} Server to authenticate client identities.
+
+.Endpoint authentication mechanisms
 
-If you include the `truststore-realm` element, the trust store must contain public certificates for all clients.
-If you do not include the `truststore-realm` element, the trust store needs only a certificate chain to verify client identities.
+If you configure {brandname} Server with a keystore only, you can use encryption in combination with any authentication mechanism.
 
-.Supported authentication mechanisms
+When you configure {brandname} Server to use a client trust store, you must configure endpoints to use the following authentication mechanisms:
 
-Trust store realms work with client-certificate authentication mechanisms:
+* Hot Rod (SASL): `EXTERNAL`
+* REST (HTTP): `CLIENT_CERT`
 
-* **SASL:** `EXTERNAL`
-* **HTTP (REST):** `CLIENT_CERT`
+[IMPORTANT]
+====
+To use perform client certificate authentication, you must also add the `require-ssl-client-auth="true"` parameter to the endpoint configuration.
```

Comment:
not sure why this is a note since it is mandatory to configure (right?)... also, the example does not have `authentication mechanisms` configured. 
also, there is another section that mention the client cert and `require-ssl-client-auth` but never said anything about the `authentication mechanisms`

Reply:
this is a reference topic so I think having it as a note works fine. there is a procedure where it is added as a step to configure the endpoint. 

for the authentication mechanism, that's configured on the client side.

Reply:
actually I see what you mean with the auth mechanisms. I'm fixing that up now, cheers.